office-365-management-api/office-365-management-activity-api-schema.md

```diff
@@ -1976,3 +1978,20 @@ The following table contain information related to AIP heartbeat events.
 |ScopeFiltersUris|Edm.String|No|The scope filter URI for the dataset in the consent operation.|
 |ScopeList|Edm.String|No|The scope group list for the dataset in the consent operation.|
 |PrivacyPolicyType|Edm.String|Yes|The privacy policy types for the dataset in the consent operation. Possible values are None and DenyList.|
+
+## Viva Goals schema
+
+The audit records for events related to Viva Goals use this schema (in addition to the [Common schema](#common-schema)).  [Search the audit log in the Security & Compliance Center](/microsoft-365/compliance/search-the-audit-log-in-security-and-compliance) details how you can search for the audit logs from the compliance portal and [Audit log activities](/microsoft-365/compliance/audit-log-activities) captures the events or activities that are related to Viva Goals. 
+
+|**Parameters**  |**Type**  |**Mandatory?**  |**Description**  |
+|---------|---------|---------|---------|
+|Detail|Edm.String |No |A description of the event or the activity that occurred on Viva Goals.|
+|Username |Edm.String </br>Term="Microsoft.Office.Audit.Schema.PIIFlag"</br>Bool="true"</br> |No |The name of the user who trigged this event|
+|UserRole |Edm.String |No |The role of the user who trigged this event within Viva Goals. This will mention if the user is an organisation admin or an owner.|
+|OrganizationName |Edm.String </br>Term="Microsoft.Office.Audit.Schema.PIIFlag" </br>Bool="true" |No |The name of the organization on Viva Goals where this event was triggered.|
+|OrganizationOwner |Edm.String </br>Term="Microsoft.Office.Audit.Schema.PIIFlag" </br>Bool="true" |No |The owner of the organization on Viva Goals in which the event occurred.|
+|OrganizationAdmins |Collection(Edm.String) </br>Term="Microsoft.Office.Audit.Schema.PIIFlag" </br>Bool="true" |No |The admins of the organization on Viva Goals where the event occurred. There can be one or more admins in the organization.|
+|UserAgent |Edm.String </br>Term="Microsoft.Office.Audit.Schema.PIIFlag" </br>Bool="true" |No |The user agent (browser details) of the user who trigged this event. This might not be present in case of system generated events.|

```

Comment:
```suggestion
|UserAgent |Edm.String </br>Term="Microsoft.Office.Audit.Schema.PIIFlag" </br>Bool="true" |No |The user agent (browser details) of the user who trigged the event. UserAgent might not be present in case of a system generated event.|

```